docs/storage.md

```diff
@@ -45,4 +62,24 @@ for asset in assets:
     asset.download()
 ```
 
+## **Pystac client**
+
+For the users who are familiar with the pystac-client library, the storage class allows to get an authenticated pystac-client for browsing the UP42 storage.
+Using the authenticated pystac_client for the UP42 account will look like:
+
+```python
+up42_pystac_client = storage.pystac_client
+```
+
+With the `up42_pystac_client`, it is possible to run the regular pystac client operations (e.g. to go through the UP42 assets AKA UP42 pystac-collections in your account). 
+
+```python
+up42_stac_collections = up42_pystac_client.get_collections()
+```
+
+Check the documentation for reference about UP42 stac definitions.
```

Comment:
maybe consider adding a hyperlink here pointing to the `UP42 STAC definitions`.

Reply:
Where do we show the actual PySTAC Client capabilities - did I miss that somehow?